### DISCUSSION TITLE
compiling with autotrace breaks ghostscript

### DISCUSSION THREAD

**Discussion by exekutive** [ImageMagick/ImageMagick#7226] [Help] [answered]
### ImageMagick version

7.1.1

### Operating system, version and so on

MacOS 14

### Description

When I compile IM with the option `--with-autotrace`, then IM compiles without gslib.
If I remove the option , ghostscript works.

Can anybody please tell me why?

[config00012.log](https://github.com/ImageMagick/ImageMagick/files/14899394/config00012.log)

**Comment by snibgo** [comment]
Your config.log includes:
```
Delegate program configuration:
  GhostPCL          None				pcl6 (unknown)
  GhostXPS          None				gxps (unknown)
  Ghostscript       None				gs (10.03.0)
```
This suggests you have GS v10.03.0. Hence I expect your IM can read PDF files. Can it?

**Comment by exekutive** [comment]
with autotrace

```
mageMagick 7.1.1-30 is configured as follows. Verify that this configuration
matches your expectations.

...
                    Option				Value
------------------------------------------------------------------------------
...
  Ghostscript lib   --with-gslib=yes			no
```

without autotrace

```
ImageMagick 7.1.1-30 is configured as follows. Verify that this configuration
matches your expectations
....

                    Option				Value
------------------------------------------------------------------------------
...
  Ghostscript lib   --with-gslib=yes			yes

```

**Comment by urban-warrior** [comment, marked as answer]
There appear to be a mistake in the `autotrace.pc` package script.  It includes @MAGICK_CFLAGS@ and @MAGICK_LIBS@ flags instead of the actual flags/libraries.  We removed these references and ImageMagick built without complaint.
